docs/guide/limitations.md

```diff
@@ -44,8 +44,9 @@ If a function is absent from the tables below, it means that no Limitations can
 |read|true|true|true|true|true|true|-|State|
 |diff|true|true|true|true|true|-|-|-|
 |view_embed|true|true|true|true|true|-|-|-|
-|create|true|true|-|true|true|-|true|Owner of Parent</br>Content Type Group of Parent</br>Content Type of Parent</br>Parent Depth|
-|edit|true|true|true|true|true|true|true|State</br>WorkflowStage|
+|create|true|true|-|true|true|-|true|Owner of Parent, Content Type Group of Parent, Content Type of Parent, Parent Depth, Translation|
```

Comment:
Let's swap `Language` with `Translation`, so `Translation` is more visible. We could even get rid of `Language`.